financialaid/models.py

```diff
@@ -10,20 +9,42 @@
 from jsonfield import JSONField
 
 from courses.models import (
-    CourseRun,
     Program,
 )
 
 
-class Tier(models.Model):
+class TimeStampedModel(models.Model):
+    """
+    Base model for create/update timestamps
+    """
+    created_on = models.DateTimeField(auto_now_add=True)  # UTC
+    updated_on = models.DateTimeField(auto_now=True)  # UTC
+
+    def update(self, **kwargs):
```

Comment:
Ah, I got some things mixed up - this update function is a helper function for saving several field changes via .update() rather than having to explicitly set attributes and save.  It's not currently used so I can take it out.

Updating the updated_on timestamp via QuerySet is implemented by subclassing the Manager and QuerySet.  Given that these models are ones that require an auditing trail, unless you disagree, I'll go ahead and add that in as a safety measure for future development.